lib/synthesize-stream.ts

```diff
@@ -14,38 +14,13 @@
  * limitations under the License
  */
 
-import { Agent, OutgoingHttpHeaders, RequestOptions } from 'http';
+import { Agent, RequestOptions } from 'http';
 import { Authenticator, qs } from 'ibm-cloud-sdk-core';
 import { Readable, ReadableOptions } from 'stream';
 import { w3cwebsocket as w3cWebSocket } from 'websocket';
+import TextToSpeechV1 = require('../text-to-speech/v1');
```

Comment:
One minor thing here, and we should probably do this in `recognize-stream` as well, I would prefer this to use object deconstruction so it's really obvious what we're importing.

```suggestion
import { SynthesizeWebSocketParams } = require('../text-to-speech/v1');
```

Since we're not importing the whole service for anything, this makes it more clear to me why the import is needed up front.

Reply:
Works for me, updated the code here and in RecognizeStream.